packages/open-truss/src/pages/config-builder-page/config-builder-context.tsx

```diff
@@ -0,0 +1,88 @@
+import get from 'lodash/get'
+import set from 'lodash/set'
+import { createContext, useState } from 'react'
+import { type FrameType, type WorkflowSpec } from '../../configuration'
+import { type YamlType, parseYaml, stringifyYaml } from '../../utils/yaml'
+
+export const CONFIG_BASE = `
+workflow:
+  version: 1
+  frameWrapper: ConfigBuilderFrameWrapper
```

Comment:
@hktouw now this is where this is configured!

Reply:
this is so cool 🚀